core/domain/feedback_domain.py

```diff
@@ -133,7 +138,10 @@ def __init__(self, full_message_id, full_thread_id, message_id, author_id,
         self.created_on = created_on
         self.last_updated = last_updated
         self.received_via_email = received_via_email
-
+        
+        raise Exception(created_on)
```

Comment:
FYI, if you want the type information, might be better to log or raise Exception on `type(created_on)` rather than `created_on`. You can also do this on your local machine.